package.json

```diff
@@ -16,4 +16,4 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject"
   }
-}
\ No newline at end of file
+}
```

Comment:
For some reason your code isn't showing! In Navigation.js, you have a <Link /> error. Check your `to=""` paths and make sure they match up exactly with your <Route /> paths in App.js. Otherwise, everything looks great.